server/repository/src/db_diesel/adjustment.rs

```diff
@@ -0,0 +1,115 @@
+use super::{adjustment::adjustments::dsl as adjustment_dsl, StorageConnection};
+
+use crate::{
+    diesel_macros::{apply_date_filter, apply_equal_filter},
+    DateFilter, EqualFilter, RepositoryError,
+};
+use diesel::prelude::*;
+
+table! {
+    adjustments (id) {
```

Comment:
This view and `replenishment` are equivalents of the consumption view, for other invoice types